docs/en/latest/terminology/secret.md

```diff
@@ -293,3 +294,56 @@ curl -i http://127.0.0.1:9080/your_route -H 'apikey: value'
 ```
 
 This will verify whether the `key-auth` plugin is correctly using the key from AWS Secrets Manager.
+
+## Use GCP Secrets Manager to manage secrets
+
+Using the GCP Secrets Manager to manage secrets means you can store the secret information in the GCP service, and reference it using a specific format of variables when configuring plugins. APISIX currently supports integration with the GCP Secrets Manager, and the supported authentication method is [OAuth 2.0](https://developers.google.com/identity/protocols/oauth2).
+
+### Reference Format
+
+```
+$secret://$manager/$id/$secret_name/$key
+```
+
+The reference format is the same as before:
+
+- manager: secrets management service, could be the HashiCorp Vault, AWS, GCP etc.
+- id: APISIX Secrets resource ID, which needs to be consistent with the one specified when adding the APISIX Secrets resource
+- secret_name: the secret name in the secrets management service
+- key: get the value of a property when the value of the secret is a JSON string
+
+### Required Parameters
```

Comment:
This portion of the doc perhaps should better be added to Admin API doc given the current structure of the apache/apisix doc since that is where the vault parameters are documented: https://apisix.apache.org/docs/apisix/admin-api/#request-body-parameters-11

cc @pottekkat for inputs as well.

Reply:
Maybe I can do it in the new PR, because I didn't add the AWS part before.
https://github.com/apache/apisix/pull/11417

Reply:
That works as well :)

Reply:
OK, I have completed：https://github.com/apache/apisix/pull/11569